src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,28 @@
-export const TodoFilter = () => (
+import { Dispatch, SetStateAction } from 'react';
+
+interface Props {
+  query: string;
+  setQuery: Dispatch<SetStateAction<string>>;
+  status: Status;
+  setStatus: Dispatch<SetStateAction<Status>>;
+}
+
+export type Status = 'all' | 'active' | 'completed';
+
+export const TodoFilter: React.FC<Props> = ({
```

Comment:
Not critical, but React.FC is not needed
https://github.com/typescript-cheatsheets/react?tab=readme-ov-file#section-2-getting-started